include/openmc/mgxs.h

```diff
@@ -85,6 +84,8 @@ class Mgxs {
   std::string name; // name of dataset, e.g., UO2
   double awr;       // atomic weight ratio
   bool fissionable; // Is this fissionable
+  bool is_isotropic {
```

Comment:
Nice. I haven't heard of anyone using angle-dependent cross sections literally ever, barring the few works from MIT about it.